docs/Roadmap.md

```diff
@@ -20,4 +20,20 @@ Any individual or organization can create an Issue on the OTP Roadmap using the
 ![choose-issue-type](images/roadmap-5.png)
 7. Fill in the necessary fields, and then click `Create`.
 <br>
-![create](images/roadmap-6.png)
\ No newline at end of file
+![create](images/roadmap-6.png)
+<be>
+
+
+A roadmap issue should follow these principles:
```

Comment:
Suggestion: "The following principles are recommended when creating a Roadmap Issue:"